indico/htdocs/js/indico/modules/categories/categories.js

```diff
@@ -111,4 +111,96 @@
             });
         }
     };
+
+    global.setupCategoryEventList = function setupCategoryEventsList() {
+        enableIfChecked('#event-management', 'input[name=event_id]', '.js-enabled-if-checked');
+
+        function createCategoryNavigator(element, data) {
+            $('<div>').categorynavigator({
+                openInDialog: true,
+                selectLeafOnly: true,
+                onAction: function(category) {
```

Comment:
This needs an extra confirmation. Moving with one click without any confirmation is not a good idea.